sdk/java/src/main/java/com/chain/api/CoreConfig.java

```diff
@@ -11,67 +11,135 @@
   public static class Info {
     public String state;
 
+    /**
+     * Whether the core has been configured.
+     */
     @SerializedName("is_configured")
     public boolean isConfigured;
 
+    /**
+     * RFC3339 timestamp reflecting when the core was configured.
+     */
     @SerializedName("configured_at")
     public Date configuredAt;
 
+    /**
+     * Whether the core is configured as a block signer.
+     */
     @SerializedName("is_signer")
     public boolean isSigner;
 
+    /**
+     * Whether the core is configured as the blockchain generator.
+     */
     @SerializedName("is_generator")
     public boolean isGenerator;
 
+    /**
+     * URL of the generator.
+     */
     @SerializedName("generator_url")
     public String generatorUrl;
 
+    /**
+     * The access token used to connect to the generator.
+     */
     @SerializedName("generator_access_token")
     public String generatorAccessToken;
 
+    /**
+     * Hash of the initial block.
+     */
     @SerializedName("blockchain_id")
     public String blockchainId;
 
+    /**
+     * Height of the blockchain in the local core.
+     */
     @SerializedName("block_height")
     public long blockHeight;
 
+    /**
+     * Height of the blockchain in the generator
+     */
     @SerializedName("generator_block_height")
     public long generatorBlockHeight;
 
+    /**
+     * RFC3339 timestamp reflecting the last time generator_block_height was updated.
+     */
     @SerializedName("generator_block_height_fetched_at")
     public Date generatorBlockHeightFetchedAt;
 
+    /**
+     * The cross-core API version supported by this core.
+     */
     @SerializedName("crosscore_rpc_version")
     public int crosscoreRpcVersion;
 
+    /**
+     * A random identifier for the core, generated during configuration.
+     */
     @SerializedName("core_id")
     public String coreId;
 
+    /**
+     * The release version of the cored binary.
+     */
     public String version;
 
+    /**
+     * Git SHA of build source.
+     */
     @SerializedName("build_commit")
     public String buildCommit;
 
+    /**
+     * Unixtime (as string) of binary build.
+     */
     @SerializedName("build_date")
     public String buildDate;
 
+    /**
+     * Features enabled or disabled in this build of Chain Core.
+     */
     @SerializedName("build_config")
     public BuildConfig buildConfig;
 
+    /**
+     * Blockchain error information.
+     */
     public Health health;
 
+    /**
+     *
```

Comment:
Oops. Meant to remove this. We don't document this key anywhere right now so I want to leave it blank.